app/client/components/VersionUpdateBanner.ts

```diff
@@ -0,0 +1,69 @@
+import {Banner, buildBannerMessage} from 'app/client/components/Banner';
+import {makeT} from 'app/client/lib/localization';
+import {localStorageJsonObs} from 'app/client/lib/localStorageObs';
+import {getGristConfig} from 'app/common/urlUtils';
+import {Disposable, dom, makeTestId, Observable} from 'grainjs';
+import {AppModel} from 'app/client/models/AppModel';
+
+const t = makeT("VersionUpdateBanner");
+const testId = makeTestId('test-version-update-banner-');
+
+interface ShowVersionUpdateBannerPrefer {
+  isShown: boolean,
```

Comment:
```suggestion
  dismissed: boolean,
```

Reply:
I think it is better to reverse the logic, and with that we even have a proper default `false`. wdyt?